docs/rules.rst

```diff
@@ -0,0 +1,622 @@
+Firewall Rules Configuration
+============================
+
+Firewall configuration in ``debops.ferm`` is done through a flexible
+definition of rules. There are a number of variables which are used to
+reference a set of default rules and can be extended by user defined
+rules. Here a description of the involved configurations should be given
+so that everyone can customize the ruleset according to individual
+requirements.
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _default_rules:
+
+Default rules
+-------------
+
+By default ``debops.ferm`` configures a number of rules as soon as a
+host is part of the ``[debops_all_hosts]`` Ansible host group. In case
+a firewall is not required or preferred this behaviour can be disabled
+by setting ``ferm__enabled: no`` in the inventory. The rules created
```

Comment:
This should be 'ferm_enabled: False'. The role checks the value as a boolean and 'no' might be converted to `True`.